lib/widgets/content.dart

```diff
@@ -1082,6 +1155,8 @@ class GlobalTime extends StatelessWidget {
             children: [
               Icon(
                 size: ambientTextStyle.fontSize!,
+                // (When GlobalTime appears in a link, it should be blue
+                // like the text.)
                 color: DefaultTextStyle.of(context).style.color!,
```

Comment:
Huh interesting. I'm not sure that's entirely intentional! But I reproduce on web, so :shrug: